charts/kaito/workspace/templates/nvidia-device-plugin-ds.yaml

```diff
@@ -28,6 +28,10 @@ spec:
                 operator: NotIn
                 values:
                 - virtual-kubelet
+              - key: accelerator
```

Comment:
I wouldn’t lean on a custom label like this unless you know that every sku, cloud (and every future cluster) will carry exactly that key/value.

Reply:
moved this setting to value.yaml